install/linux/vendor-pymupdf.py

```diff
@@ -28,7 +28,7 @@ def main():
     )
```

Comment:
Reading this, I wonder if we should rename the `vendor-pymupdf` to something specifically targeting debian-based distributions.

Reply:
After discussion, we've decided to go with `debian-vendor-pymupdf.py`